includes/admin/settings.php

```diff
@@ -498,7 +523,7 @@ function fastwc_onboarding_url_content() {
  */
 function fastwc_get_option_or_set_default( $option, $default ) {
 	$val = get_option( $option );
-	if ( $val ) {
+	if ( false === $val ) {
```

Comment:
is this correct? seems like we are either returning `false` or the default now

Reply:
➕  probably should be !== ?

Reply:
It should be `!==`. This is now fixed.